spring-batch-infrastructure/src/main/java/org/springframework/batch/item/ExecutionContext.java

```diff
@@ -371,6 +371,16 @@ public Object remove(String key) {
 		return this.map.remove(key);
 	}
 
+	/**
+	 * Removes all mappings from this context.
+	 * After this method call, the context will be empty.
+	 *
+	 * @see java.util.Map#clear()
+	 */
+	public void clear() {
+		this.map.clear();
```

Comment:
I found that this code base was worked on in 2008, and there's a few places where I could do some code cleansing, such as removing `this` keyword where it's clear that it doesn't need to be used.

However, code cleansing has a slightly different purpose than the current PR work, so I'll work on it as a separate PR if needed.